.github/workflows/MSBuild.yml

```diff
@@ -12,7 +13,7 @@ env:
 
 jobs:
   build:
-    runs-on: windows-latest
+    runs-on: windows-2022
```

Comment:
I would assume latest is 2022? Or is that still 2019?

Reply:
Yes windows-latest is currently windows-2022. To ensure there are no implicit changes of the runner "windows-2022" might be the better option